plugins/nginx-vhosts/commands

```diff
@@ -50,7 +50,7 @@ case "$1" in
     fi
 
     DOKKU_APP_CIDS=($(get_app_container_ids $APP))
-    docker cp "${DOKKU_APP_CID[0]}:/app/nginx.conf.template" "$APP_NGINX_TEMPLATE" 2> /dev/null || true
+    docker cp "${DOKKU_APP_CIDS[0]}:/app/nginx.conf.template" "$DOKKU_ROOT/$APP/" 2> /dev/null || true
```

Comment:
:+1: on fixing `$DOKKU_APP_CIDS`. I missed that one in review. Any reason to change from using `$APP_NGINX_TEMPLATE`? I like variable re-use. :smile:

@josegonzalez to your question, in docker 1.6.2 at least, the destination does not *have* to be a path to a file.